src/setup_jest.ts

```diff
@@ -0,0 +1,14 @@
+const originalConsoleError = console.error
+
+// Look into removing this on the next React update to 0.16.
+console.error = (message?: any, ...optionalParams: any[]) => {
+  if (
+    typeof message === "string" &&
+    (message.includes("Accessing PropTypes via the main React package") ||
+      message.includes("React.createClass is no longer supported."))
+  ) {
+    // NOOP
+  } else {
+    originalConsoleError(message)
```

Comment:
Are you sure you want to overwrite native code? These warnings will disappear in time and _have_ to be gone by the time React 16 comes around. Also, I think the proper way to shim this to preserve existing functionality is

```js
const originalConsoleError = console.error.bind(console)

function error() {
  const message = arguments[0]

  if (
    typeof message === "string" &&
    (message.includes("Accessing PropTypes via the main React package") ||
      message.includes("React.createClass is no longer supported."))
  ) {
    // NOOP
  } else {
    originalConsoleError(...arguments)
  }
}

console.error = error
```